apps/studio/components/interfaces/HomeNew/ActivityStats.tsx

```diff
@@ -0,0 +1,131 @@
+  })
+  const latestMigration = useMemo(() => (migrationsData ?? [])[0], [migrationsData])
+
+  const { data: backupsData, isLoading: isLoadingBackups } = useBackupsQuery({
+    projectRef: project?.ref,
+  })
+  const latestBackup = useMemo(() => {
+    const list = backupsData?.backups ?? []
+    if (list.length === 0) return undefined
+    return list
+      .slice()
+      .sort((a, b) => new Date(b.inserted_at).valueOf() - new Date(a.inserted_at).valueOf())[0]
+  }, [backupsData])
+
+  return (
+    <div className="@container">
+      <div className="grid grid-cols-2 xl:flex xl:gap-10 gap-4 flex-wrap">
+        <div className="block">
+          <p className="heading-meta text-foreground-light mb-1">Status</p>
+          <ServiceStatus />
+        </div>
+
+        <Link className="block" href={`/project/${ref}/database/migrations`}>
+          <h4 className="heading-meta text-foreground-light mb-1">Last migration</h4>
+
+          <div className={cn('h-[34px] flex items-center capitalize-sentence')}>
+            {isLoadingMigrations ? (
+              <Skeleton className="h-6 w-24" />
+            ) : latestMigration ? (
+              <TimestampInfo
+                className="text-base"
+                label={dayjs(latestMigration.version, 'YYYYMMDDHHmmss').fromNow()}
+                utcTimestamp={dayjs(latestMigration.version, 'YYYYMMDDHHmmss').toISOString()}
+              />
+            ) : (
+              <p className="text-foreground-lighter">No migrations</p>
+            )}
+          </div>
+        </Link>
+
+        <Link className="block" href={`/project/${ref}/database/backups/scheduled`}>
+          <h4 className="heading-meta text-foreground-light mb-1">Last backup</h4>
+
+          <div className={cn('h-[34px] flex items-center capitalize-sentence')}>
+            {isLoadingBackups ? (
+              <Skeleton className="h-6 w-24" />
+            ) : backupsData?.pitr_enabled ? (
+              <p>PITR enabled</p>
+            ) : latestBackup ? (
+              <TimestampInfo
+                className="text-base"
+                label={dayjs(latestBackup.inserted_at).fromNow()}
+                utcTimestamp={latestBackup.inserted_at}
+              />
+            ) : (
+              <p className="text-foreground-lighter">No backups</p>
+            )}
+          </div>
+        </Link>
+
+        <Link className="block" href={`/project/${ref}/branches`}>
+          <h4 className="heading-meta text-foreground-light mb-1">
+            {isDefaultProject ? 'Recent branch' : 'Branch Created'}
+          </h4>
+
+          <div className="text-foreground truncate h-[34px] flex items-center capitalize-sentence">
+            {isLoadingBranches ? (
+              <Skeleton className="h-6 w-24" />
+            ) : isDefaultProject ? (
+              <div className="flex items-center gap-2">
+                <GitBranch size={16} strokeWidth={1.5} className="text-foreground-lighter" />
```

Comment:
Minor low-stakes suggestion: Make the `GitBranch` icon condition conditional so the empty state matches the other rows (no icon). Or visa versa; add icons to the others too. Right now it sticks out a bit.

<img width="1228" height="350" alt="Image" src="https://github.com/user-attachments/assets/aaae5fff-2966-4d87-a8cf-25905a891af0" />